shared/profile/wot-author/index.tsx

```diff
@@ -356,49 +547,55 @@ const WotModal = (props: WotModalProps) => {
   )
 }
 
-const VerificationChoice = (props: {
-  children?: React.ReactNode
+/* eslint-disable sort-keys */
+const verificationTypeColors = {
+  in_person: Styles.globalColors.greenDark,
+  video: '#56fff5',
+  audio: Styles.globalColors.blueLight,
+  proofs: Styles.globalColors.blueLight,
+  other_chat: Styles.globalColors.yellow,
+  familiar: Styles.globalColors.yellow,
+  other: Styles.globalColors.yellowDark,
+}
+/* eslint-enable sort-keys */
+
+const verificationTypeText = (
+  verificationType: WebOfTrustVerificationType,
   voucheeUsername: string
-  verificationType: WebOfTrustVerificationType
-  selected: boolean
-  onSelect: () => void
-}) => {
-  let text: React.ReactNode = 'Do not choose this option'
-  let color: string = Styles.globalColors.white
-  switch (props.verificationType) {
+): React.ReactNode => {
+  switch (verificationType) {
     case 'in_person':
-      text = (
+      return (
         <>
-          {props.voucheeUsername} told me their username <Kb.Text type="BodyBold">in person</Kb.Text>
+          {voucheeUsername} told me their username <Kb.Text type="BodyBold">in person</Kb.Text>
         </>
       )
-      color = Styles.globalColors.greenDark
-      break
     case 'video':
-      text = `${props.voucheeUsername} told me their username over video`
-      color = '#56fff5'
-      break
+      return `${voucheeUsername} told me their username over video`
     case 'audio':
-      text = `${props.voucheeUsername} told me their username over audio`
-      color = Styles.globalColors.blueLight
-      break
+      return `${voucheeUsername} told me their username over audio`
     case 'proofs':
-      text = `I know one of ${props.voucheeUsername}'s proofs`
-      color = Styles.globalColors.blueLight
-      break
+      return `I know one of ${voucheeUsername}'s proofs`
     case 'other_chat':
-      text = `${props.voucheeUsername} texted me their username`
-      color = Styles.globalColors.yellow
-      break
+      return `${voucheeUsername} texted me their username`
     case 'familiar':
-      text = 'We are longtime Keybase friends'
-      color = Styles.globalColors.yellow
-      break
+      return 'We are longtime Keybase friends'
     case 'other':
-      text = 'Other'
-      color = Styles.globalColors.yellowDark
-      break
+      return 'Other'
+    default:
+      return `<Unknown verification type: ${verificationType}>`
```

Comment:
Why brackets?

Reply:
Also should this be an error bar?

Reply:
Brackets removed. According to typescript it will never happen. I just don't want it to crash even if typescript is wrong by way of bugs or if the service doesn't play nice. But going all the way to an error bar would be too much refactoring for a situation typescript says will never happen.